src/org/usfirst/frc/team694/robot/commands/auton/SameSideSwitchAutonCommand.java

```diff
@@ -0,0 +1,39 @@
+package org.usfirst.frc.team694.robot.commands.auton;
+
+import org.usfirst.frc.team694.robot.FieldMap;
+import org.usfirst.frc.team694.robot.commands.DrivetrainRotateDegreesPIDCommand;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+/**
+ *
+ */
+public class SameSideSwitchAutonCommand extends CommandGroup {
+    public double speed = 0.5; 
+    public SameSideSwitchAutonCommand() {
+        addSequential(new DrivetrainMoveToLineCommand(FieldMap.DISTANCE_FROM_ALLIANCE_STATION_TO_AUTO_LINE, speed));
+        addSequential(new DrivetrainMoveInchesEncoderCommand(FieldMap.DISTANCE_TO_TRAVEL_BEFORE_FIRST_TURN_FROM_AUTO_LINE, speed));
+        addSequential(new DrivetrainRotateDegreesPIDCommand(90));
+        addSequential(new DrivetrainMoveToLineCommand(FieldMap , speed);
+        addSequential(new DrivetrainMoveInchesEncoderCommand(FieldMap. , speed));
+        addSequential(new DrivetrainRotateDegreesPIDCommand(90)); 
+        addSequential(new DrivetrainMoveInchesEncoderCommand(FieldMap., speed));
+        addSequential(new push);
```

Comment:
Will not compile. I'd suggest removing this from the branch and putting it elsewhere (in another branch maybe? This auton may not be up to date)